templates/nginx.j2

```diff
@@ -1,5 +1,5 @@
 upstream channels-backend {
-    server localhost:8000;
+    server unix://{{ openwisp2_path }}/daphne0.sock fail_timeout=0;
```

Comment:
this one points to `daphne0.sock`, are you sure this difference is not a problem?